Src/WitsmlExplorer.Api/Models/MudLog.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Collections.Generic;
+
+namespace WitsmlExplorer.Api.Models
+{
+    public class MudLog
+    {
+        public string WellUid { get; set; }
+        public string WellboreUid { get; set; }
+        public string Uid { get; set; }
+        public string WellName { get; set; }
+        public string WellboreName { get; set; }
+        public string Name { get; set; }
+        public string ObjectGrowing { get; set; }
+        public string MudLogCompany { get; set; }
+        public string MudLogEngineers { get; set; }
+        public string StartMd { get; set; }
+        public string EndMd { get; set; }
+        public DateTime? DateTimeCreation { get; set; }
+        public DateTime? DateTimeLastChange { get; set; }
```

Comment:
Are these (`DateTimeCreation`, `DateTimeLastChange`) the same properties that we find in `CommonData.cs` and `CommonTime.cs` ? If so I think we might want to consolidate them in one place?